app/src/lib/stores/app-store.ts

```diff
@@ -1607,6 +1608,41 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
       return { workingDirectory, selectedFileIDs, diff }
     })
+
+    const inConflictedMerge = status.workingDirectory.files.some(f => {
+      return (
+        f.status === AppFileStatus.Conflicted ||
+        f.status === AppFileStatus.Resolved
+      )
+    })
+
+    // if any files are conflicted or resolved,
+    // we trigger the conflict resolution flow
+    // (if we're not already in it)
+    if (
+      status.currentBranch !== undefined &&
+      inConflictedMerge &&
+      (this.currentPopup === null ||
```

Comment:
Are we able to extract this part of the check out to something more readable, and indicate what we're looking for here?